### ISSUE TITLE
No style applied to: archive/index.html

### ISSUE THREAD

**Issue by uaqben** [octopress/genesis-theme#35]
Using a brand new Octopress (with multilingual plugin; in case it impacts).

The layout and style of the theme works great, except when one browses to:

 `http://127.0.0.1:4000/archive/index.html`

Where it renders old-school style; and also seems to ignore the settings of `_config.yml`, as it shows:

* default `My Octopress Blog` title in the header
* default `Copyright © 2015 - Powered by Octopress` in the footer

**Comment by imathis** [comment]
Yeah this is still very much a work in progress. I'm in the process of reworking the layout and making the footer not stupid. Thanks for trying it out and finding this one!

**Comment by uaqben** [comment]
Cool. I was thinking: Would it help you to pull (some of) the content of octopress.org and put it in a repo managed by Octopress 3.0?

I feel it could help to have "real site" content to finish and polish all Octopress features, such as multilingual which I'm having difficulties with, and the new theme... I'd be happy to provide pull requests with content translation to another language (es/fr).

just an idea - let me know ;)
